modules/apps/analytics/analytics-batch-export-import-impl/src/main/java/com/liferay/analytics/batch/exportimport/internal/dto/v1_0/converter/DXPEntityDTOConverterImpl.java

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.analytics.dxp.entity.rest.internal.dto.v1_0.converter;
+package com.liferay.analytics.batch.exportimport.internal.dto.v1_0.converter;
```

Comment:
Hey @MrJohn1911, Could you please add in the commit message the reason why you are moving this class? Thx